server/application/application.go

```diff
@@ -86,6 +86,18 @@ func (s *Server) Create(ctx context.Context, a *appv1.Application) (*appv1.Appli
 	return out, err
 }
 
+// GetManifests returns application manifests
+func (s *Server) GetManifests(ctx context.Context, q *ApplicationQuery) (*repository.ManifestResponse, error) {
+	// CompareAppState compares application spec and real app state using KSonnet
+	app, err := s.Get(ctx, q)
+	if err != nil {
+		return nil, err
+	}
+	repo := s.getRepo(ctx, app.Spec.Source.RepoURL)
+
+	return controller.GetManifest(s.repoClientset, repo, app)
```

Comment:
application server shouldn't have dependency on controller code. In fact, `Server.appComparator` should be removed, now that alex moved the sync logic into controller.

Reply:
This was my most recent change.  Reverting `HEAD`.